openlibrary/plugins/openlibrary/api.py

```diff
@@ -634,3 +635,17 @@ def POST(self, work_id: str):
             ),
             content_type="application/json",
         )
+
+
+class search_opds(delegate.page):
+    path = "/search"
+    encode = 'opds'
```

Comment:
😂😂😂 I mean I think this new endpoint is _great_, don't get me wrong! But I did also like the html endpoint we had before! Maybe we have both? 😋

```suggestion
    encoding = 'opds'
```